portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,116 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>get to know Dez</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <img src="/desmondsimages/headerpic.png" 
+             alt="Dez Image #1" 
+             style="float:left;"
+              class="image1">
+        <h1>
+             get to know Dez
+        </h1>
+            <p class="leftpad1">
+                <strong>NAME</strong>: Desmond Talton <br><br>
+                <strong>COLLEGE</strong>: The Georgia Institute of Technology <br><br>
+                <strong>MAJOR</strong>: Computer Science <br><br>
```

Comment:
Indentation seems a bit off, `<p>` should be align to `<h1>`, can you please make sure to fix other indentation as well(if any).

Reply:
Fixed the indentation throughout the html file.